kernel/CMakeLists.txt

```diff
@@ -94,6 +94,16 @@ list(APPEND kernel_files
 endif()
 
 add_library(kernel ${kernel_files})
+target_link_libraries(kernel c_picolibc)
+target_link_libraries(kernel c_common)
+
+add_library(c INTERFACE IMPORTED GLOBAL)
```

Comment:
We can't use `add_library(c` here -- the minimal C library and the picolibc module both need to build this object, and they get processed ahead of the kernel?

Reply:
doesn't matter, it's an imported lib, not a regular lib.
And when linking it to the kernel, then anyone linking to the kernel inherits the information.